manifests/init.pp

```diff
@@ -165,6 +170,10 @@
   $kadmind_enable = true,
   $kadmind_acls = { "*/admin@$realm" => '*' },
 
+  $host_ticket_cache_ccname = "/tmp/krb5cc.puppet",
```

Comment:
Predictable file names in temp folders are bad. A temporary file needs to have a nonpredictable name and be securely created using something like the mktemp command. If the filename must be static and shared around, then it should be in a root-owned folder.